src/main/java/core/basesyntax/service/FruitTransaction.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax.service;
+
+public class FruitTransaction {
```

Comment:
The class `FruitTransaction` should be in its own file named `FruitTransaction.java` in the `model` package to follow the project structure guidelines.